js/ethereum-utils/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import * as address from './address';
+import * as payloads from './payloads';
+import * as signature from './signature';
+import * as signatureDefinitions from './signature.definitions';
+
+export * from './payloads';
+export * from './signature';
+export * from './signature.definitions';
+export * from './address';
+
+export default { ...payloads, ...address, ...signatureDefinitions, ...signature };
```

Comment:
this flattens all the definitions and functions which if duplicated might cause some collisions in future. If you have some better organization ideas please let me know.

Reply:
I think this works.

I was able to do both of these, which is good:

- `import { getEip712BrowserRequestAddKeyData } from "@frequency-chain/ethereum-utils"`
- `import frqEth from "@frequency-chain/ethereum-utils"`